src/objstore.cc

```diff
@@ -273,7 +273,7 @@ void ObjStoreService::process_gets_for_objref(ObjRef objref) {
       ObjHandle& elem = memory_[objref].first;
       RAY_CHECK(send_queues_[get_queue_[i].first].send(&item.first), "error sending over IPC");
       // Remove the get task from the queue
-      std::swap(get_queue_[i], get_queue_[get_queue_.size() - 1]);
+      using std::swap; swap(get_queue_[i], get_queue_[get_queue_.size() - 1]);
```

Comment:
Yes it's ugly but what is currently there is a subtly wrong way to call `swap`, which only happens to achieve the intended purpose here because the data types are pretty benign. It's a C++ idiosyncrasy that `swap` is a nonmember function that needs to be found via something called "argument-dependent lookup" (ADL), because in reality the point of `swap` is that it may be implemented in an entirely different namespace and hence the call need not resolve to `std::swap` at all. It's not an aesthetic change, it's a semantic change and it should always be done like this.